advocacy_docs/edb-postgres-ai/ai-accelerator/pipelines-overview.mdx

```diff
@@ -1,46 +1,46 @@
 ---
 title: "AI Accelerator - Pipelines overview"
-navTitle: "Pipelines Overview"
+navTitle: "Pipelines overview"
 description: "Where to start with AI Accelerator Pipelines."
 prevNext: True
 redirects:
 - /purl/aidb/pipelinesoverview
 ---
 
-As part of the EDB Postgres AI's AI Accelerator Pipelines abstracts away the complexity of working with AI data. It transforms Postgres into a powerful platform for AI data management, as it combines vector search from PGvector with automation for complex AI workflows.
+As part of the EDB Postgres AI - AI Accelerator, Pipelines abstracts away the complexity of working with AI data. It transforms Postgres into a powerful platform for AI data management, combining vector search from PGvector with automation for complex AI workflows.
```

Comment:
```suggestion
As part of EDB Postgres AI - AI Accelerator, Pipelines abstracts away the complexity of working with AI data. It transforms Postgres into a powerful platform for AI data management, combining vector search from PGvector with automation for complex AI workflows.
```

Reply:
I'm still not completely happy with this this first sentence. I'd like to separate EDB Postgres AI from this product name (AI Accelerator Pipelines?? Still unclear, see previous comments). Do we need to say EDB Postgres AI here anyhow? I'd be okay with As part of AI Accelerator, Pipelines...
Not sure if this was instead meant to be:
As part of the EDB Postgres AI platform, AI Accelerator Pipelines...